### DISCUSSION TITLE
Neoforge

### DISCUSSION THREAD

**Discussion by zjedi** [meza/minecraft-mod-manager#547] [Q&A] [answered]
option mentions forge, but does it work with neoforge too? It looks like neoforge is the way to go these days.

**Comment by meza** [comment, marked as answer]
Hey, yes, it's already in the @next release! https://github.com/meza/minecraft-mod-manager/tree/next?tab=readme-ov-file#loaders

You can give it a go here https://github.com/meza/minecraft-mod-manager/releases/tag/v2.0.0-next.8
